site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -101,9 +123,15 @@ ifdef::samlv2_idp_initiated[]
 This field must be set to `SAMLv2IdPInitiated`.
 endif::[]
 
+[field]#identityProvider.uniqueIdClaim# [type]#[String]# [optional]#Optional#::
```

Comment:
What do we do if you omit this optional parameter? This explains what we do the value but not what this parameter does to help us find that value.